app/models/maintenance_tasks/progress.rb

```diff
@@ -73,5 +74,9 @@ def estimatable?
     def over_total?
       @run.tick_count > @run.tick_total
     end
+
+    def ticks_in_words(amount, unit = 'item')
+      "#{amount} #{unit.pluralize(amount)}"
```

Comment:
There's a `pluralize` helper that does that: https://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-pluralize

Reply:
Nice, forgot this was a thing! 🙌

Reply:
Oh, i remember why we didn't use this before - it's from `action_view`. Are we okay with including the `TextHelper` module?